server/graphql/v1/mutations/collectives.js

```diff
@@ -94,6 +94,7 @@ export async function createCollective(_, args, req) {
 
     collectiveData.currency = collectiveData.currency || hostCollective.currency;
     collectiveData.hostFeePercent = hostCollective.hostFeePercent;
+    collectiveData.platformFeePercent = hostCollective.platformFeePercent;
```

Comment:
Only if it's set, non null.

Reply:
Maybe you should handle that within `addHost`?

Reply:
Nice catch, we should add the not null conditional.
I don't think we should move this to `addHost` because I think we should keep this behaviour only for newly created collectives. If we move this to `addHost` we might end up changing platformFee for collectives that are changing host. WDYT?

Reply:
@kewitz I think that's actually a desirable effect. Many Collectives will actually be created without host and it will be added later.